packages/help-center/jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
 	preset: '../../test/packages/jest-preset.js',
+	setupFiles: [ '<rootDir>/jestSetup.ts' ],
 	testEnvironment: 'jsdom',
+	moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx', 'json', 'node' ],
+	transformIgnorePatterns: [ 'node_modules/(?!react-native|react-navigation)/' ],
```

Comment:
I don't entirely understand what's happening here, but this resolves an error with file transformations I was seeing when trying to run jest. Maybe someone else can chime in with more info?